### PR TITLE
Feature/survey

### PR DESCRIPTION
**Pull request by everyplace**
## Intro
This PR provides a sample implementation for Surveys, but also accomplishes a few other things:

- Makes the prompt invocation examples a bit more generic. Survey and Newsletter prompts share the same code, just a different configuration.
- Adds a few overrides per-environment, notably being able to specify a version of `swg.js` other than the default.
- Adds the ability to configure Google Analytics on the global template.

## Preview
- See the ephemeral build in [PR 39](https://pr-39-dot-reader-revenue-demo.ue.r.appspot.com/survey/manual)
- Test the actual surveys in [nightly](https://nightly-dot-reader-revenue-demo.ue.r.appspot.com/survey/manual)

## History
Tasks that were completed since the first draft of this PR:
- [x] Update the `cloudbuild.yaml` configuration to allow for setting the `GTAG_PROPERTY_ID` and `GTAG_DEBUG_MODE` values
- [x] Update the newsletter example code to refer to the updated generic prompt invocation semantics
- [x] Create the sample code for the Surveys page

### CODE REVIEW

File: app/content/survey-manual.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<script async
+  subscriptions-control="manual"
+  type="application/javascript"
+  src="https://news.google.com/swg/js/v1/swg-qual.js">
```
Comment (everyplace) [inline app/content/survey-manual.md:4]: - [ ] Need to change this back to use SWG_OVERRIDE

File: app/content/overview.md [hunk 2]
```diff
@@ -68,6 +68,14 @@ PUBLICATION_ID=publisher-center-ppid.google.com
 OAUTH_CLIENT_ID=abcd-1234.apps.googleusercontent.com
 OAUTH_CLIENT_SECRET=secret-abc-1234
 GOOGLE_SITE_VERIFICATION=public-abc-1234
+
+# Google Analytics
+GTAG_PROPERTY_ID: G-1234567890  #Configure gTag Id
```
Comment (Wayne-X) [inline app/content/overview.md:73]: mega-nit, space after # before comment e.g. "# Configure gTag Id"

File: app/content/survey-manual.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+### Invoke Survey Prompts
+
+To configure survey prompts, `swg.js` must first be configured on the page.
+These examples show using the initialization of the library in `manual` mode, but the
+APIs are also available in automatic mode. 
+
+### Get the prompt instance to display
+
+To invoke a survey prompt, a publisher must use the `configurationId` provided by
+Google in response to submitting a prompt configuration. Publishers use the
```
Comment (Wayne-X) [inline app/content/survey-manual.md:94]: Publishers get the available configurationId by using the ... (?)

**Review by Wayne-X — APPROVED** [review]

File: app/content/overview.md (hunk 2 above)
Comment (everyplace) [reply app/content/overview.md:73]: @Wayne-X oh actually the `:` is a typo. Will push a fix...

File: app/content/survey-manual.md (hunk 3 above)
Comment (mhkawano) [reply app/content/survey-manual.md:94]: `subscriptions.getAvailableInterventions()`? There's some example code right under here

File: app/content/survey-manual.md [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+    }
+});
+```
+
+### Handle the response
+
+The `onResult` callback will include information on the configuration used
+to create the prompt, as well as the newsletter subscriber's information. The `configurationId` matches the `configurationId provided to the publisher from Google, in response to the configuration authored by the publisher in the initial step.
+
+!!! note `onResult()` callback format is variable.
```
Comment (mhkawano) [inline app/content/survey-manual.md:132]: The type is documented here, and I think we're gonna try pretty hard to not change it https://github.com/subscriptions-project/swg-js/blob/main/src/api/available-intervention.ts:InterventionResult

File: app/content/survey-manual.md (hunk 3 above)
Comment (everyplace) [reply app/content/survey-manual.md:94]: I added a note re: Publisher Center page.

File: app/content/survey-manual.md [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+
+This complete example accomplishes the following:
+
+1. Initializes `swg.js` library in manual mode.
+2. When the library is ready, use the `49c12712-9750-4571-8c67-96722561c13a` configurationId to request a prompt to display.  
+3. When the button is clicked, display the prompt.
+4. Store the results of a successful prompt with the sample `PromptPersistence()` library.
+
+!!! note `PromptPersistence()` is an example implementation.
+In a production environment, a publisher would use the prompt response to send data to their own
```
Comment (mhkawano) [inline app/content/survey-manual.md:148]: "to their own account" -> "to their Google Analytics account"? I don't think newsletters are sent to GA, but they can be downloaded in Publisher Center

Comment (everyplace) [reply app/content/survey-manual.md:149]: Added a clarification re: CRM as the destination.

File: app/content/survey-manual.md (hunk 4 above)
Comment (mhkawano) [reply app/content/survey-manual.md]: ops, yes!

**Review by mhkawano — APPROVED** [review]
LGTM

**Pull request merged**